Dockerfile.pgadmin

```diff
@@ -0,0 +1,39 @@
+FROM python:2.7-alpine
```

Comment:
I would prefer to use another template. This one has a lot more stars and Python v3. https://hub.docker.com/r/fenglc/pgadmin4/~/dockerfile/